manifests/1.14/deploy/vsphere-csi-node-ds.yaml

```diff
@@ -43,6 +43,10 @@ spec:
           image: vmware/vsphere-block-csi-driver:v1.0.0
           imagePullPolicy: "Always"
           env:
+            - name: NODE_NAME
```

Comment:
Is this env var read anywhere? Couldn't find anything by searching  `NODE_NAME`.